framework/aster-frame/src/task/mod.rs

```diff
@@ -2,13 +2,15 @@
 
 //! Tasks are the unit of code execution.
 
+mod atomic;
 mod priority;
 mod processor;
 mod scheduler;
 #[allow(clippy::module_inception)]
 mod task;
 
 pub use self::{
+    atomic::{enter_atomic_mode, might_break_atomic_mode, AtomicModeGuard},
```

Comment:
Should we expose these methods as public APIs of the framework? Is there any use for these APIs outside of the framework?